deploy/ansible/roles/filebeat/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+shipper_name: default
+env: test
+project_name: iroha-bench
+app_name: iroha-la
+logstash_url: elk.soramitsu.co.jp:5055
```

Comment:
Please add comments to iroha/deploy/ansible/roles/filebeat/defaults/main.yml values, and comment out soramitsu ELK address or remove it.